Docs/README.md

```diff
@@ -20,20 +20,21 @@ It simply updates the file (with all existing data unchanged) to be in a given c
 
 
 Other presets available:
-Presets:
-	bt.709          Rec. ITU-R BT.709-6
-	srgb-linear     linear-light sRGB
-	srgb            IEC 61966-2-1 sRGB
-	bt.2020-10-bit  Rec. ITU-R BT.2020-2 (10-bit system)
-	bt.2020-12-bit  Rec. ITU-R BT.2020-2 (12-bit system)
-	bt.2100-pq      Rec. ITU-R BT.2100-2 perceptual quantization (PQ) system
-	bt.2100-hlg     Rec. ITU-R BT.2100-2 hybrid log-gamma (HLG) system
-	dci-p3          SMPTE RP 431-2 with SMPTE ST 428-1 D-Cinema Distribution Master (DCI-P3)
-	display-p3      Display P3
-	p3-d65-pq       P3-D65 PQ
+| Preset | Specification |
+|--------|---------------|
+| bt.709         | Rec. ITU-R BT.709-6 |
+| srgb-linear    | linear-light sRGB |
+| srgb           | IEC 61966-2-1 sRGB |
+| bt.2020-10-bit | Rec. ITU-R BT.2020-2 (10-bit system) |
+| bt.2020-12-bit | Rec. ITU-R BT.2020-2 (12-bit system) |
+| bt.2100-pq     | Rec. ITU-R BT.2100-2 perceptual quantization (PQ) system |
+| bt.2100-hlg    | Rec. ITU-R BT.2100-2 hybrid log-gamma (HLG) system |
+| dci-p3         | SMPTE RP 431-2 with SMPTE ST 428-1 D-Cinema Distribution Master (DCI-P3) |
+| display-p3     | Display P3 |
+| p3-d65-pq      | P3-D65 PQ |
```

Comment:
Good idea, to put it in a table.
I'm torn on an idea. GitHub allows you to pad out the table delimiters with spaces and -. You did this to align the first column.

Should we align throughout? I think we should.
So like:
```
| Preset         | Specification                          |
|----------------|----------------------------------------|
| bt.709         | Rec. ITU-R BT.709-6                    |
...
```

Reply:
Done